Assets/Scripts/FarmlandBlock.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//Czy potrzebujemy interfejsu "interakcja"? Farm block moze go implementowac
+//Potrzebujemy klasy nadrzednej "Block"? Czy beda inne bloki?
```

Comment:
Niezbyt rozumiem te dwa pytania, Co to interfejs? Nie rozumiem też pytania "Czy będą inne bloki?".

Reply:
Ok, w takim razie ten fragment kodu na razie zawiesimy, przynajmniej do momentu kiedy wprowadzimy interfejsy do Twojego repertuaru programistycznego.
Przykładem interfejsu mogą być wszystkie meble w simsach - każdy z nich jest inną klasą, ale wszystkie implementują interfejs "interakcja" który pozwala im reagować na poczynania gracza, zapewniając grę o posiadaniu funkcji "Akcja()".
Powinno to być wyjaśnione w pdf'ie od profesora Beziny, tuż po obiektach. Zagadnienie bardzo podobne do dziedziczenia ;)
Na razie dodałem "blok uprawny" który pozwala na sadzenie roślinek. Czy będziemy chcieli dodać bloczki które będą robiły coś innego? Np blok pozwalający na uzupełnianie konewki? Albo blok który ma nie robić nic i po prostu stać i ładnie wyglądać?

Reply:
W takim razie blok może działać tak samo jak interfejs? Bo to co zrozumiałam to fajnie by było mieć taki interfejs, bo większość rzeczy raczej będzie możliwe do podniesienia albo ogółem interakcji z tą rzeczy. Dalej niezbyt rozumiem też na czym mają polegać "bloki". Mają to być swego rodzaju takie kratki na których coś będzie i będzie można coś robić na nim? A jeśli tak to nie będzie to w sumie miało takiego samego działania jak interfejs?